src/helper.ts

```diff
@@ -0,0 +1,16 @@
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import usersFromServer from './api/users';
+
+export function getUser(userId: number): User | null {
```

Comment:
Use more descriptive name.
```suggestion
export function getUserById(userId: number): User | null {
```